docusaurus.config.js

```diff
@@ -50,6 +50,9 @@ const config = {
           editUrl:
             'https://github.com/ed-fi-alliance-oss/ed-fi-alliance-oss.github.io/tree/main/blog/',
         },
+        googleTagManager: {
+          containerId: "GTM-KGR2977"
+        },
```

Comment:
@roberthunterjr I think you also need to [install the plugin](https://www.docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-tag-manager)

Reply:
Thanks! There was a note in the plugin docs that said

>If you use the preset @docusaurus/preset-classic, you don't need to install this plugin as a dependency.

This leads me to believe that the GTM can be configured directly in the preset config, and without installing the additional dependency.

Reply:
OK, and now I see the important note: "This plugin is always inactive in development and only active in production to avoid polluting the analytics statistics."